src/icons/NotAllowedIcon.tsx

```diff
@@ -12,12 +14,12 @@ export const Error: React.FC<React.SVGProps<SVGSVGElement>> = (props) => (
     <circle cx="20" cy="20" r="16" fill="#F5FAFB" />
     <circle cx="20" cy="20" r="20" fill="white" />
     <path
-      fillRule="evenodd"
-      clipRule="evenodd"
+      fill-rule="evenodd"
```

Comment:
I think react will complain about this not being camelCased

Reply:
Changed